apps/total-typescript/src/templates/workshop-template.tsx

```diff
@@ -164,16 +184,19 @@ const Header: React.FC<{workshop: Module}> = ({workshop}) => {
   )
 }
 
-const WorkshopSectionNavigator: React.FC<{workshop: Module}> = ({workshop}) => {
+const WorkshopSectionNavigator: React.FC<{
+  workshop: Module
```

Comment:
instead, just calling again. no need to pass in as props, you can call the router as many times as needed